package.json

```diff
@@ -66,7 +66,7 @@
   "devDependencies": {
     "@edx/frontend-build": "^5.6.11",
     "@edx/frontend-platform": "1.8.4",
-    "@edx/paragon": "13.17.0",
+    "@edx/paragon": "19.14.1",
```

Comment:
I could not find any relevant breaking changes in 13->19. Plus, the learning MFE has been on 19 for a while while using this lib and we haven't heard reports of issues yet.